examples/api/lib/material/progress_indicator/circular_progress_indicator.1.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Flutter code sample for [CircularProgressIndicator].
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ProgressIndicatorApp());
+
+class ProgressIndicatorApp extends StatelessWidget {
+  const ProgressIndicatorApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(useMaterial3: true, colorSchemeSeed: const Color(0xff6750a4)),
+      home: const ProgressIndicatorExample(),
+    );
+  }
+}
+
+class ProgressIndicatorExample extends StatefulWidget {
+  const ProgressIndicatorExample({super.key});
+
+  @override
+  State<ProgressIndicatorExample> createState() => _ProgressIndicatorExampleState();
+}
+
+/// [AnimationController]s can be created with `vsync: this` because of
```

Comment:
Maybe move this comment down to just above the `vsync:` line?

Reply:
Sure!